src/synthesizer/emission_models/dust/emission.py

```diff
@@ -358,40 +377,44 @@ def _lnu(self, nu: unyt_array) -> unyt_array:
                 The unnormalised spectral luminosity density.
 
         """
+        # Ensure we have units on nu
+        if not has_units(nu):
+            raise exceptions.InconsistentArguments(
+                f"Frequency must be a given with units. (type(nu)={type(nu)})"
+            )
 
-        return nu**self.emissivity * planck(nu, self.temperature)
```

Comment:
This is actually correct. That is the functional form. The units are supposedly corrected at normalisation. One thing we could do make it (nu/nu_0)**emissivity to make it unitless?

Reply:
See for the different parameterisation definitions of the IR emission in: https://ui.adsabs.harvard.edu/abs/2012MNRAS.425.3094C/abstract

Reply:
Sure, we could do that. The pure nu multiplication isn't dimensionally consistent so we should avoid it. Seems a weird parametrisation though. Can't look at the paper right now but I'll check it out.

Reply:
It is defined as a proportionality in their definition, so probably they did not care about the units.

Reply:
That'd do it 😂. We can just arbitrarily divide by Hz to maintain the previous behaviour?